src/test/java/com/monumental/services/integrationtest/MonumentServiceIntegrationTests.java

```diff
@@ -0,0 +1,298 @@
+package com.monumental.services.integrationtest;
+
+import com.monumental.models.Monument;
+import com.monumental.services.MonumentService;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.test.context.SpringBootTest;
+import org.springframework.test.annotation.DirtiesContext;
+import org.springframework.test.context.junit4.SpringJUnit4ClassRunner;
+
+import java.util.ArrayList;
+import java.util.List;
+
+import static org.junit.Assert.*;
+
+@SpringBootTest
+@RunWith(SpringJUnit4ClassRunner.class)
+@DirtiesContext(classMode = DirtiesContext.ClassMode.BEFORE_EACH_TEST_METHOD)
```

Comment:
This Spring annotation makes sure that the database is reset before each test method runs so that we can know what is in there and what IDs to expect on inserts.